File: 1. Conditional Problems/Conditional Problems in Python/p8.py
#  write a python program to input any character and check whether it is alphabet, digit, or special character.

ch = input("Enter Your Character: ")

if (ch.isalpha()):
    print(f"{ch} is an Alphbet")
elif (ch.isdigit()):
    print(f"{ch} is a Digit")
else:
    print(f"{ch} is a special symbol")
    

File: 1. Conditional Problems/Condional Problems in C/p8.c
// write a C program to input any character and check whether it is alphabet, digit, or special character.

#include <stdio.h>

int main()
{
    char ch;
    printf("Enter your character: ");
    scanf("%c", &ch);

    if ((ch >= 'a' && ch <= 'z') || (ch >= 'A' && ch <= 'Z')) {
        printf("%c is an Alphabet",ch);
    }
    else if (ch >= '0' && ch <= '9') {
        printf("%c is a Digit",ch);
    }
    else {
        printf("%c is a Special symbol",ch);
    }
}